runtime/src/chplsys.c

```diff
@@ -52,9 +52,65 @@
 #include <unistd.h>
 
 
-#ifndef chplGetPageSize
-#define chplGetPageSize() sysconf(_SC_PAGE_SIZE)
+size_t chpl_getSysPageSize(void) {
+  static size_t pageSize = 0;
+
+  //
+  // No conflict management.  We assume this is set once, early, before
```

Comment:
Do you need conflict management for anything here?  If multiple threads call this, isn't the worst that will happen that pagesize might get set multiple times?